contracts/ColonyExpenditure.sol

```diff
@@ -125,20 +125,20 @@ contract ColonyExpenditure is ColonyStorage {
     emit ExpenditureSkillSet(_id, _slot, _skillId);
   }
 
-  function setExpenditurePayoutScalar(
+  function setExpenditurePayoutModifier(
     uint256 _permissionDomainId,
     uint256 _childSkillIndex,
     uint256 _id,
     uint256 _slot,
-    uint256 _payoutScalar
+    uint256 _payoutModifier
   )
     public
     stoppable
     authDomain(_permissionDomainId, _childSkillIndex, expenditures[_id].domainId)
   {
-    require(_payoutScalar <= MAX_PAYOUT_SCALAR, "colony-expenditure-payout-scalar-too-large");
+    require(_payoutModifier <= MAX_PAYOUT_MODIFIER, "colony-expenditure-payout-scalar-too-large");
 
-    expenditureSlots[_id][_slot].payoutScalar = packPayoutScalar(_payoutScalar);
+    expenditureSlots[_id][_slot].payoutModifier = packPayoutModifier(_payoutModifier);
```

Comment:
I still don't like this, but that's because I didn't really explain myself properly. Now, I'm still setting the expenditure payout modifier, passing in `_payoutModifier`, but I end up setting `expenditureSlots[_id][_slot].payoutModifier` to a value that is not `_payoutModifier`. But if I `getExpenditureSlot`, then I get the value I passed in. That's the weird behaviour I don't like, which isn't sorted.

My motiviation to rename the variable was so that the modifier could be stored unaltered (as a value between -1e18 and 1e18, with a default of zero), and then the 'packing' (which I would actually just call a calculation, at that point) could be done when it was used during payout to modify the 'default' payout scalar of 1e18.